### PR TITLE
Remove var

### PR DESCRIPTION
**Pull request by raphamorim**
Ref: #11736

### CODE REVIEW

File: packages/react-dom/src/__tests__/ReactDOMInput-test.js [hunk 1]
```diff
@@ -141,19 +141,14 @@ describe('ReactDOMInput', () => {
               value="lion"
               onChange={e => this.change(e.target.value)}
             />
-            <input
-              type="checkbox"
-              ref={n => (this.b = n)}
-              checked={true}
-              onChange={() => {}}
-            />
+            <input type="checkbox" ref={n => (this.b = n)} checked={true} />
```
Comment (gaearon) [inline packages/react-dom/src/__tests__/ReactDOMInput-test.js]: This is undoing a recently merged change.

Comment (raphamorim) [reply packages/react-dom/src/__tests__/ReactDOMInput-test.js]: damn, you're right. I'll solve it, can you continue review the other files?
I had conflict only with this file and the ReactFiber of reconciler package

**Comment by raphamorim** [comment]
Hm, I discovery other inconsistencies @gaearon (my mistake 😞 )
I'll solve it and update here.

**Comment by raphamorim** [comment]
Fixed @gaearon

**Comment by gaearon** [comment]
Sorry, I merged a few things and it's out of date again. Maybe we can split this further?

**Comment by raphamorim** [comment]
Sure, I break in other Pull Requests.

**Pull request closed without merging**